### PR TITLE
Feature/#11 rls

### PR DESCRIPTION
**Pull request by GalochkinKirov**

### CODE REVIEW

File: server/migrations/20190325001122-rolesCol.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  up: async function (queryInterface) {
+    await queryInterface.sequelize.query(`
+        ALTER TABLE "AgentsChannels" ADD COLUMN "roles" VARCHAR(8)[];
+        UPDATE "AgentsChannels" SET "roles" = ARRAY['viewer', 'admin'];
```
Comment (OAGr) [inline server/migrations/20190325001122-rolesCol.js]: Why didn't you use the regular way of modifying tables here? Also, is this a required attribute? (And if so, make sure defaults are set, probably to viewer)

Comment (GalochkinKirov) [reply server/migrations/20190325001122-rolesCol.js]: Thanks, fixed.

File: server/src/middlewares/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      await channel(root, args, context, info);
+      await agentsChannels(root, args, context, info);
+      return await resolve(root, args, context, info);
+    },
+  }
+};
+
+module.exports = {
+  middlewares,
+};
```
Comment (OAGr) [inline server/src/middlewares/index.js]: How are the middlewares used in this story? The connection isn't clear to me.

Comment (GalochkinKirov) [reply server/src/middlewares/index.js]: Middlewares prepare context of users requests. Authorizers use the context. That is all.

File: server/src/data/channels-data.js [hunk 3]
```diff
@@ -22,14 +22,18 @@ class ChannelsData extends DataBase {
...
+      return Promise.reject(new Error('Channel exists.'));
     }
+    channel = await models.Channel.create({
+      ...input,
+      creatorId: user.agentId,
+    });
+    await this.agentsChannelsData.createOne(
+      channel.id,
+      user.agentId,
+      models.AgentsChannels.ROLE.ADMIN,
```
Comment (OAGr) [inline server/src/data/channels-data.js]: Can this be done in an ``afterCreate`` callback in the Channel model layer? That seems like where I would more expect it.

File: server/src/middlewares/index.js (hunk 2 above)
Comment (OAGr) [reply server/src/middlewares/index.js]: Ah, thanks.

File: server/src/data/channels-data.js (hunk 3 above)
Comment (GalochkinKirov) [reply server/src/data/channels-data.js]: No, according to our strategy models do not know logic of application work. They only represents data from DB.

**Pull request merged**